docs/docs/features/power-shaping.md

```diff
@@ -0,0 +1,52 @@
+# Power Shaping
+
+To give consumer chains more flexibility in choosing their validator set, Interchain Security offers
+several "power shaping" mechanisms for consumer chains.
+
+These are:
+1) **Capping the size of the validator set**: The consumer chain can specify a maximum number of validators it
```

Comment:
Does a capped validator receive rewards based on its full power or the capped power?

Reply:
Capped power, see https://github.com/cosmos/interchain-security/blob/ph/docs-fix/x/ccv/provider/keeper/distribution.go#L153